doc/nrf/gs_installing.rst

```diff
@@ -55,6 +55,61 @@ The installation process is different depending on your operating system.
 
       Also see :ref:`zephyr:mac-setup-alts` for additional information.
 
+If you are interested in building `Connected Home over IP`_ applications, install also the `GN`_ meta-build system to generate files for `Ninja`_.
```

Comment:
Maybe reword a bit, since the user probably doesn't care about generating Ninja files, but wants to use the NCS. ;)
```suggestion
If you are interested in building `Connected Home over IP`_ applications, install also the `GN`_ meta-build system to generate the `Ninja`_ files that the |NCS| uses.
```